lib/screens/stops/stop.ex

```diff
@@ -310,6 +310,19 @@ defmodule Screens.Stops.Stop do
     end
   end
 
+  @callback fetch_routes_for_stops(stop_ids :: String.t() | [String.t()]) ::
+              {:ok, [Route.t()]} | {:error, term()}
+  def fetch_routes_for_stops(stop_ids) do
```

Comment:
We already have a similar `Route.serving_stop` function; maybe these could be combined. I think it may be better to go with that function's approach — just asking for routes with a `filter[stop]` and letting the API decide what it means for a route to serve a stop. At least, that seems like the behavior we'd want for fetching alerts? (to not limit ourselves to "canonical" service only)